rest-openehr/src/main/java/org/ehrbase/rest/openehr/audit/support/OpenEhrAuditMessageBuilder.java

```diff
@@ -0,0 +1,48 @@
+package org.ehrbase.rest.openehr.audit.support;
```

Comment:
Please add a license header to new files.

Reply:
Done